diff -up squid-5.8/src/cf.data.pre.config squid-5.8/src/cf.data.pre
--- squid-5.8/src/cf.data.pre.config	2023-02-28 11:01:10.000000000 +0100
+++ squid-5.8/src/cf.data.pre	2023-03-04 15:45:04.377280208 +0100
@@ -5313,7 +5313,7 @@ DOC_END
 
 NAME: logfile_rotate
 TYPE: int
-DEFAULT: 10
+DEFAULT: 0
 LOC: Config.Log.rotateNumber
 DOC_START
 	Specifies the default number of logfile rotations to make when you
@@ -7215,7 +7215,7 @@ COMMENT_END
 
 NAME: cache_mgr
 TYPE: string
-DEFAULT: webmaster
+DEFAULT: root
 LOC: Config.adminEmail
 DOC_START
 	Email-address of local cache manager who will receive
@@ -8600,10 +8600,10 @@ COMMENT_END
 NAME: icon_directory
 TYPE: string
 LOC: Config.icons.directory
-DEFAULT: @DEFAULT_ICON_DIR@
+DEFAULT: /usr/share/squid/icons
 DOC_START
 	Where the icons are stored. These are normally kept in
-	@DEFAULT_ICON_DIR@
+	/usr/share/squid/icons
 DOC_END
 
 NAME: global_internal_static
@@ -8646,8 +8646,10 @@ DEFAULT_DOC: Send error pages in the cli
 DOC_START
 	If you wish to create your own versions of the default
 	error files to customize them to suit your company copy
-	the error/template files to another directory and point
-	this tag at them.
+	/usr/share/squid/errors contains sets of error files
+        in different languages. The default error directory
+        is /etc/squid/errors, which is a link to one of these
+        error sets.
 
 	WARNING: This option will disable multi-language support
 	         on error pages if used.
