diff -up squid-3.2.5/lib/libTrie/configure.ac.automake-1_13 squid-3.2.5/lib/libTrie/configure.ac
--- squid-3.2.5/lib/libTrie/configure.ac.automake-1_13	2012-12-10 10:53:26.000000000 +0100
+++ squid-3.2.5/lib/libTrie/configure.ac	2013-01-06 02:00:03.927852277 +0100
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 AC_INIT(libTrie, 0.1, rbtcollins@squid-cache.org)
 AC_CONFIG_SRCDIR([src/Trie.cc])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_AUX_DIR(cfgaux)
 
 enable_inline=${enable_inline:=yes}
