--- squid-3.4.9/doc/Makefile.am.datadir	2014-10-31 10:36:43.000000000 +0100
+++ squid-3.4.9/doc/Makefile.am	2014-12-08 15:54:23.485442559 +0100
@@ -3,7 +3,7 @@ SUBDIRS = manuals
 
 DEFAULT_CONFIG_FILE = $(sysconfdir)/squid.conf
 DEFAULT_MIME_TABLE      = $(sysconfdir)/mime.conf
-DEFAULT_ERROR_DIR       = $(datadir)/errors
+DEFAULT_ERROR_DIR       = $(datadir)/squid/errors/@ERR_DEFAULT_LANGUAGE@
 
 EXTRA_DIST = \
 	debug-sections.txt
--- squid-3.4.9/errors/Makefile.am.datadir	2014-12-08 15:54:23.485442559 +0100
+++ squid-3.4.9/errors/Makefile.am	2014-12-08 16:09:55.842786488 +0100
@@ -1,7 +1,7 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-errordir = $(datadir)/errors
+errordir = $(datadir)/squid/errors
 
 ##dist-hook eveything
 
--- squid-3.4.9/icons/Makefile.am.datadir	2014-10-31 10:36:43.000000000 +0100
+++ squid-3.4.9/icons/Makefile.am	2014-12-08 15:54:23.485442559 +0100
@@ -1,7 +1,7 @@

 include $(top_srcdir)/icons/icon.list
 
-icondir = $(datadir)/icons/silk
+icondir = $(datadir)/squid/icons/silk
 icon_DATA = $(ICONS)
 EXTRA_DIST = $(ICONS) icon.list SN.png
 DISTCLEANFILES = 
--- squid-3.4.9/src/Makefile.am.datadir	2014-10-31 10:36:43.000000000 +0100
+++ squid-3.4.9/src/Makefile.am	2014-12-08 15:54:23.486442546 +0100
@@ -936,8 +936,8 @@ DEFAULT_PINGER		= $(libexecdir)/`echo pi
 DEFAULT_UNLINKD		= $(libexecdir)/`echo unlinkd | sed '$(transform);s/$$/$(EXEEXT)/'`
 DEFAULT_LOGFILED	= $(libexecdir)/`echo log_file_daemon | sed '$(transform);s/$$/$(EXEEXT)/'`
 DEFAULT_DISKD		= $(libexecdir)/`echo diskd | sed '$(transform);s/$$/$(EXEEXT)/'`
-DEFAULT_ICON_DIR	= $(datadir)/icons
-DEFAULT_ERROR_DIR	= $(datadir)/errors
+DEFAULT_ICON_DIR	= $(datadir)/squid/icons
+DEFAULT_ERROR_DIR	= $(datadir)/squid/errors/@ERR_DEFAULT_LANGUAGE@
 
 # Make location configure settings available to the code
 DEFS += -DDEFAULT_CONFIG_FILE=\"$(DEFAULT_CONFIG_FILE)\" -DDEFAULT_SQUID_DATA_DIR=\"$(datadir)\" -DDEFAULT_SQUID_CONFIG_DIR=\"$(sysconfdir)\"
